package.json

```diff
@@ -10,6 +10,7 @@
   },
   "dependencies": {
     "@apollo/react-hooks": "^3.1.5",
+    "@babel/runtime": "^7.0.0",
```

Comment:
There's an auto-update from Axios-hooks which is @babel/runtime needs to be installed, otherwise, it'll show an error.

<img width="637" alt="Screen Shot 2565-01-18 at 22 27 22" src="https://user-images.githubusercontent.com/12908129/149967224-6616b549-9c92-4427-8ad0-673d8b895f65.png">